lexicons/app/bsky/feed/getAuthorFeed.json

```diff
@@ -4,12 +4,12 @@
   "defs": {
     "main": {
       "type": "query",
-      "description": "A view of a user's feed.",
+      "description": "A view of an actor's feed.",
       "parameters": {
         "type": "params",
-        "required": ["author"],
+        "required": ["actor"],
```

Comment:
I took a bit of liberty here & changed it to `actor` even tho it wasn't ambiguous like `user`. Let me know if you disagree